BizHawk.Client.Common/movie/tasproj/IStateManager.cs

```diff
@@ -36,8 +35,7 @@ public interface IStateManager : IDisposable
 
 		int Count { get; }
 
-		// TODO: rename to Last
-		int LastStatedFrame { get; }
+		int Last { get; }
```

Comment:
The only unpleasant thing here is that it's now way harder to search where these are called.

Reply:
find all references?